### PR TITLE
[checkpoint_harmony_endpoint] Tidy the Server URL var, screenshots

### PR DESCRIPTION
**Pull request by chrisberkhout**
## Proposed commit message

```
[checkpoint_harmony_endpoint] Tidy the Server URL var, screenshots

Don't use the EU server as a default for base_url.

Update the Add Integration screenshot and move it after the dashboard
screenshot.
```

## Checklist

- [ ] I have reviewed [tips for building integrations](https://github.com/elastic/integrations/blob/main/docs/tips_for_building_integrations.md) and this pull request is aligned with them.
- [ ] I have verified that all data streams collect metrics or logs.
- [ ] I have added an entry to my package's `changelog.yml` file.
- [ ] I have verified that Kibana version constraints are current according to [guidelines](https://github.com/elastic/elastic-package/blob/master/docs/howto/stack_version_support.md#when-to-update-the-condition).
- [ ] I have verified that any added dashboard complies with Kibana's [Dashboard good practices](https://docs.elastic.dev/ux-guidelines/data-viz/dashboard-good-practices)

### CODE REVIEW

File: packages/checkpoint_harmony_endpoint/changelog.yml [hunk 1]
```diff
@@ -1,6 +1,11 @@
 # newer versions go on top
+- version: "0.1.1"
+  changes:
+    - description: Put the dashboard screenshot first
+      type: bugfix
+      link: https://github.com/elastic/integrations/pull/10652
```
Comment (chemamartinez) [inline packages/checkpoint_harmony_endpoint/changelog.yml:6]:
```suggestion
      link: https://github.com/elastic/integrations/pull/11652
```

**Review by chemamartinez — APPROVED** [review]
LGTM! Just a small typo in the PR link.

File: packages/checkpoint_harmony_endpoint/changelog.yml (hunk 1 above)
Comment (chrisberkhout) [reply packages/checkpoint_harmony_endpoint/changelog.yml:6]: Good catch. Thanks!

**Comment by chrisberkhout** [comment]
@chemamartinez Sorry, one more change in here. The default Server URL was the EU server. I removed the default and put examples for US and EU in the description.

**Pull request merged**